src/js/plugin/external/PluginHelper.js

```
@@ -1,27 +1,41 @@
 import PluginDispatcher from "./PluginDispatcher";
+import PluginEvents from "./PluginEvents";
 
-var PluginHelper = {
-  callAction: function (action, data) {
+export default class PluginHelper {
+
+  constructor(pluginId) {
```

Should we throw an error here, if the pluginId isn't given?
What do you think?